shared/.pronto_spell.pws

```diff
@@ -18,6 +18,7 @@ Phonelib
 Pragma
 Procfile
 Rakefile
+Reoon
```

Comment:
Let’s lowercase this and please don’t update version in PR, I do this in master when publish to RubyGems.